docs/sources/why-use-continuous-profiling/_index.md

```diff
@@ -0,0 +1,88 @@
+menuTitle: "Why Use Continuous Profiling?"
+description: "Discover the benefits of continuous profiling and its role in modern application performance analysis."
+weight: 20
+keywords:
+  - pyroscope
+  - phlare
+  - continuous profiling
+  - flamegraphs
+---
+
+# Why Use Continuous Profiling?
+
+![Visual comparison between traditional and continuous profiling](#)
+
+Continuous profiling is more than just a performance analysis tool; it's a crucial component in modern software development and operations. It goes past traditional profiling techniques by providing ongoing, in-depth insights into application performance. 
+
+Continuous profiling goes past the ephemeral, localized nature of traditional profiling (which historically has been more similar to "console.log" or "print statement" debugging) to a structured, centralized approach allows for effective use in production environments. Put more simply, Pyroscope takes you from a bunch of flamegraph files on your desktop to a database where you can query and analyze production data in a structured way.
+
+Pyroscope in particular, offers you the flexibility to either visualize more "traditional" adhoc data or evolve your applications observability tooling to include more "modern" continuous profiling capabilities.
+
+## What is Continuous Profiling?
+
+Continuous profiling is a systematic method of collecting and analyzing performance data from production systems.
+
+Traditionally, profiling has been used more as an ad-hoc debugging tool. While used in many languages, particularly in Go and Java many are used to running a benchmark tool locally and getting a pprof file in go or maybe ssh'ing into a misbehaving prod instance and pulling a flamegraph from a JFR file in Java. This is great for debugging but not so great for production.
+
+Continuous profiling is a much more modern approach which is safer and more scalable for production environments. It makes use of low overhead sampling to collect profiles from production systems and stores them in a database for later analysis. This allows you to get a much more holistic view of your application and how it behaves in production.
+
+## The Core Benefits of Continuous Profiling
+
+![Diagram showing 3 benefits of continuous profiling](#)
+
+Why prioritize continuous profiling? Here are the key reasons:
+1. **In-Depth Code Insights:** It provides granular, line-level insights into how application code utilizes resources, offering the most detailed view of application performance.
+2. **Complements Other Observability Tools:** Continuous profiling fills critical gaps left by metrics, logs, and tracing, creating a more comprehensive observability strategy.
+3. **Proactive Performance Optimization:** Regular profiling enables teams to proactively identify and resolve performance bottlenecks, leading to more efficient and reliable applications.
+
+## Business Impact of Continuous Profiling
+
+![Infographic illustrating key business benefits](#)
```

Comment:
<img width="564" alt="image" src="https://github.com/grafana/pyroscope/assets/23323466/62909fb0-c389-4c4c-a0c3-a178f4f71fc8">

Reply:
Instructions for adding images: https://grafana.com/docs/writers-toolkit/write/image-guidelines/